<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioDialog</class>
 <widget class="QDialog" name="ScenarioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="2">
    <widget class="QPushButton" name="writeScenarioPushButton">
     <property name="text">
      <string>Write Scenario to File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="readScenarioPushButton">
     <property name="text">
      <string>Read Scenario from File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="bodySettingsBox">
     <property name="title">
      <string>Body Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="planetCountLabel">
        <property name="text">
         <string>Number of Planets</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="planetCountNumber">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="planetSelectedLabel">
        <property name="text">
         <string>Selected Planet</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="planetSelectedComboBox">
        <property name="maxCount">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="planetNameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QFrame" name="posVelFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="4">
          <widget class="QLabel" name="zLabel">
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="planetPositionZ">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>-100000000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="velocityLabel">
           <property name="text">
            <string>Velocity</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="planetPositionY">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>-100000000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="positionUnitLabel">
           <property name="text">
            <string>Length Unit</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="planetVelocityY">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>-100000000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>-100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="xLabel">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QDoubleSpinBox" name="planetVelocityZ">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>-100000000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="velocityUnitLabel">
           <property name="text">
            <string>Length/Time Unit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="yLabel">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="planetPositionX">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>-100000000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="planetVelocityX">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>-100000000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="positionLabel">
           <property name="text">
            <string>Position</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="planetMassLabel">
        <property name="text">
         <string>Mass</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="planetMassSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000000000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="planetRadiusLabel">
        <property name="text">
         <string>Radius</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="planetRadiusSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="planetColorLabel">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="planetColorComboBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="planetNameLine">
        <property name="text">
         <string>Planet X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="loadScenarioPushButton">
     <property name="text">
      <string>Load Scenario into simulator</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scenario Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="3">
       <widget class="QLabel" name="massUnitLabel">
        <property name="text">
         <string>Mass Units</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dtLabel">
        <property name="text">
         <string>Delta Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="gravConstSpinBox">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>6.674080000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Power</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Symbol</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="gravConstLabel">
        <property name="text">
         <string>Gravitational Constant</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QLineEdit" name="timeUnitSymbolLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>s</string>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Long Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="iterPerDataLabel">
        <property name="text">
         <string>Iterations per Data Point</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QComboBox" name="lengthUnitComboBox"/>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="text">
         <string>Default Scenario</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="lengthUnitSymbolLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>m</string>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Scenario Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="lengthUnitLabel">
        <property name="text">
         <string>Length Units</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLineEdit" name="massUnitSymbolLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>kg</string>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="timeUnitLabel">
        <property name="text">
         <string>Time Units</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="endTimeLabel">
        <property name="text">
         <string>End Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="gravConstPowerSpinBox">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="value">
         <number>-11</number>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QComboBox" name="timeUnitComboBox"/>
      </item>
      <item row="5" column="4">
       <widget class="QComboBox" name="massUnitComboBox"/>
      </item>
      <item row="2" column="3">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0" rowspan="3">
          <widget class="QLabel" name="gravConstValue">
           <property name="text">
            <string>6.6e-11</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="timeLabel">
           <property name="text">
            <string>s^2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="massLabel">
           <property name="text">
            <string>kg * </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLabel" name="lengthLabel">
           <property name="text">
            <string>m^3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="endTimeSpinBox">
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QSpinBox" name="iterPerDataSpinBox">
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QFrame" name="dtFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QSpinBox" name="dtSpinBox">
           <property name="minimum">
            <number>-50</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="twoToThePowerOfLabel">
           <property name="text">
            <string>2 to the power of</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="dtLabelUnit">
           <property name="text">
            <string>s</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>gravConstSpinBox</tabstop>
  <tabstop>endTimeSpinBox</tabstop>
  <tabstop>iterPerDataSpinBox</tabstop>
  <tabstop>planetCountNumber</tabstop>
  <tabstop>planetSelectedComboBox</tabstop>
  <tabstop>planetNameLine</tabstop>
  <tabstop>planetPositionX</tabstop>
  <tabstop>planetPositionY</tabstop>
  <tabstop>planetPositionZ</tabstop>
  <tabstop>planetVelocityX</tabstop>
  <tabstop>planetVelocityY</tabstop>
  <tabstop>planetVelocityZ</tabstop>
  <tabstop>planetMassSpinBox</tabstop>
  <tabstop>planetRadiusSpinBox</tabstop>
  <tabstop>planetColorComboBox</tabstop>
  <tabstop>readScenarioPushButton</tabstop>
  <tabstop>loadScenarioPushButton</tabstop>
  <tabstop>writeScenarioPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
